src/odb/src/db/dbProperty.cpp

```diff
@@ -182,21 +162,35 @@ dbIStream& operator>>(dbIStream& stream, _dbProperty& prop)
   stream >> prop._next;
   stream >> prop._owner;
 
+  uint boolean;
+  int integer;
+  char* char_string;
+  std::string string_property;
+  double double_property;
+
   switch (prop._flags._type) {
     case DB_BOOL_PROP:
-      stream >> prop._value._bool_val;
+      stream >> boolean;
+      prop._value = boolean;
       break;
 
     case DB_INT_PROP:
-      stream >> prop._value._int_val;
+      stream >> integer;
+      prop._value = integer;
       break;
 
     case DB_STRING_PROP:
-      stream >> prop._value._str_val;
+      stream >> char_string;
+      prop._value = "";
```

Comment:
move to else block

Reply:
Not sure I agree. Seems like simpler control flow to just set the empty string up front since that's the default case.

i.e. no matter what we're setting the variant to a string.

Reply:
It is slightly inefficient to set and reset the string but I doubt it matters that much.  If you prefer it this way I don't care strongly.